src/sentry/web/client_config.py

```diff
@@ -186,6 +186,9 @@ def get_client_config(request=None):
         "demoMode": settings.DEMO_MODE,
         "enableAnalytics": settings.ENABLE_ANALYTICS,
         "validateSUForm": getattr(settings, "VALIDATE_SUPERUSER_ACCESS_CATEGORY_AND_REASON", False),
+        # TODO: to be implemented in the future when customer domain work evolves
+        "apiUrl": options.get("system.url-prefix"),
```

Comment:
@markstory Do you remember if this should be `sentryUrl`? I can't recall if we settled on either `apiUrl` or `sentryUrl`.